dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyTreeFormatter.java

```diff
@@ -98,14 +98,15 @@ private static void formatDependencyPathTree(
   }
 
   /**
-   * Builds ListMultiMap that represents a Maven dependency tree of parent-children relationship.
-   * The root node is represented as an empty {@link DependencyPath} and its children are
-   * the values for the root node.
+   * Builds ListMultimap that represents a Maven dependency tree of parent-children relationship.
    *
    * @param dependencyPaths dependency path instances without assuming any order
-   * @return ListMultiMap representing a Maven dependency tree
+   * @return ListMultimap representing a Maven dependency tree of parent-children relationship. The
+   *     {@link DependencyPath} representing the root Maven artifact is available by querying the
+   *     tree with an empty {@link DependencyPath} instance. For a {@code node} in the returned
```

Comment:
Since a ListMultimap's values are ordered, why isn't the root simply the first value in the map? That would be easier to retrieve and seem more obvious.